assets/home/notices.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * WordPress dependencies
+ */
+import { RawHTML } from '@wordpress/element';
+
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import SenseiCircleLogo from '../images/sensei-circle-logo.svg';
+import Link from './link';
+
+/**
+ * Component to render an action of a given notice.
+ *
+ * @param {Object} props        Component props.
+ * @param {Object} props.action The action to render.
+ */
+const NoticeAction = ( { action } ) => {
+	if ( ! action || ! action.label || ! action.url ) {
+		return null;
+	}
+
+	const isPrimary = action.primary ?? true;
+
+	const buttonClass = isPrimary ? 'button-primary' : 'button-secondary';
+	return (
+		<a
+			href={ action.url }
+			target={ action.target ?? '_blank' }
```

Comment:
I think we should default this to `_self` for internal actions. For example, for activations and updates they should stay on the same page.

Reply:
Hey @jom! Thanks for the review.

The initial implementation was setting this as `_self`, but then I changed to `_blank` because of [this comment](https://github.com/Automattic/sensei/issues/5949#issuecomment-1281195349)  made by @Luchadores, which mentioned that he prefers to open all the links in a new tab.

Naturally, I can change back if you still prefer to go that way, as I'm not sure if this also applies to notices. But I think it's worth mentioning. :)

Reply:
I can see it for external links, but I think it is a pretty serious anti-pattern for internal links because it forces me to open in a new tab vs giving me the power to do it myself with a CMD key etc. In this case, I think we definitely usually want to because we their action will change the state of the notice so we want them to come back and see the notice cleared.

Reply:
Okay, good points. Changed the default in bfab7e0f0d8855f3269c96bee4c6be175acee9b9 - Thanks!